coderd/notifications/dispatch/smtp/html.gotmpl

```diff
@@ -1,27 +1,32 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>{{ .Labels._subject }}</title>
-</head>
-<body style="font-family: Arial, sans-serif; background-color: #1d1d20; margin: 0; padding: 0;">
-<div style="max-width: 600px; margin: 20px auto; background-color: #3f556d; border: 1px solid #34495E; padding: 20px; border-radius: 8px;">
-    <div style="text-align: center; padding: 20px 0; border-bottom: 1px solid #34495E;">
-        <img width="215" height="47" src="https://coder.com/logo-wide-white.png"/>
-    </div>
-    <div style="padding: 20px; color: #ECF0F1; line-height: 1.6;">
-        <h1 style="color: #ECF0F1;">{{ .Labels._subject }}</h1>
+  </head>
+  <body style="margin: 0; padding: 0; font-family: -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif; color: #020617; background: #f8fafc;">
+    <div style="max-width: 600px; margin: 20px auto; padding: 60px; border: 1px solid #e2e8f0; border-radius: 8px; background-color: #fff; text-align: left; font-size: 14px; line-height: 1.5;">
+      <div style="text-align: center;">
+        <img src="https://coder.com/coder-logo-300x300.png" alt="Coder Logo" style="width: 60px;" />
```

Comment:
I think we probably want a logo with our name in it, though.

Reply:
Hm... I like the clean aspect of the icon, but I don't have a strong opinion. Why do you prefer the full logo instead of the icon?

Reply:
Our logo is not very recognisable.

Reply:
Maybe for people who are not using the product, it could be less recognizable. For example, Vercel's logo is a generic triangle, but their users recognize it because they use the product, and I think it's the same for Coder. In the UI, for example, we only use the icon, and I think people know it's Coder. My preference would be to use only the icon for visual consistency and a cleaner look. What do you think, @stirby?

Reply:
I think it's important that folks know that these emails are coming from coder so that they don't trash them or mark them as spam.

Reply:
In the footer, we include the Coder company name and the deployment URL. In my opinion, the subject is the most important information, and as I mentioned before, the notification emails will be sent to Coder users who are already familiar with our branding and context. However, since this isn't a blocker, I'll replace the icon with the horizontal logo that includes the name, as you suggested. We can revisit this discussion later if we think it adds value. Does that sound like a good plan?

Reply:
SGTM 👍

Reply:
<img width="1240" alt="Screenshot 2024-08-09 at 11 02 21" src="https://github.com/user-attachments/assets/00213fbc-6051-449f-8faf-1c08d88ff8a0">